dev/bots/test.dart

```diff
@@ -588,6 +588,7 @@ Future<void> _flutterBuild(
   await runCommand(flutter, <String>[
     'build',
     platformBuildName,
+    if (verifyCaching) '--performance-measurement-file=perf.json',
```

Comment:
The test runs the build command twice and then verifies that the targets no-op on the second run.

Previously, only the second run passed a `--performance-measurement-file` flag. This flag updates the generated Xcode config, which invalidated the check dev dependencies target.

Now, both runs pass the `--performance-measurement-file` flag. This ensures the generated Xcode config doesn't change on the second run, and ensures all targets no-op.

 created a performance measurement file 

This performance measurement file is passed to `flutter assemble` through the `